### PR TITLE
Add Korean docs folder and Korean content

### LINKED ISSUE #176: Add Korean Language Support in Documentation
/area documentation

**What would you like to be added**:
Add Korean as a supported language for falco documentations.

**Why is this needed**:
To help Korean users understand falco better.

### PR DESCRIPTION
**Pull request by onesolpark**
**What type of PR is this?**
/kind content
/kind translation
/area documentation

**What this PR does / why we need it**:
Helps Korean users understand Falco better :)

**Which issue(s) this PR fixes**:

Fixes #176 

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by onesolpark** [comment]
/assign @radhikapc
Added sign off message :)

**Comment by jmyung** [comment]
/assign @jmyung

May I review this? 😃

File: config.yaml [hunk 1]
```diff
@@ -259,7 +259,91 @@ languages:
...
+            text: 어플리케이션에 대한 이상감지에 대한 보안 룰을 풍부한 컨텍스트와 유연성을 지닌 엔진을 이용해 생성할 수 있습니다.
+          - title: 위험 감소
+            icon: box.png
+            text: Falco를 존재하는 보안 워크플로우나 프로세스에 플러그인 하여 보안 위반 사항에 대한 즉각적인 응답을 받을 수 있습니다.
+          - title: 최신 룰셋 활용
+            icon: scalable.png
+            text: 커뮤니티에서 소싱된 이상 감지 및 CVE 취약점에 대한 알림을 받을 수 있습니다.
+      syscalls:
+        title: 시스템 콜(System calls)을 활용한 확장된 가시성
+        summary: Falco는 보안 메카니즘인 Extended Berkeley Packet Filter (eBPF)를 효율적으로 사용하여 시스템 콜(system call)을 캡처하여 더 많은 가시성을 제공합니다. 쿠버네티스 어플리케이션 컨텍스트와 쿠버네티스 API audit 이벤트를 더해 관리자는 어떤 사용자가 무슨 행위를 한지 파악할 수 있습니다.
```
Comment (jmyung) [inline config.yaml]:
```suggestion
        summary: Falco는 보안 메커니즘인 Extended Berkeley Packet Filter (eBPF)를 효율적으로 사용하여 시스템 콜(system call)을 캡처하여 더 많은 가시성을 제공합니다. 쿠버네티스 애플리케이션 컨텍스트와 쿠버네티스 API 감사(audit) 이벤트를 추가하여 팀은 어떤 사용자가 어떤 행위를 했는지 파악할 수 있습니다.
```
- 매커니즘, 애플리케이션 : 위에서 언급드린 내용입니다.
- 감사(audit) : 한영 병기 표기를 제안드립니다.
- 좀더 자연스럽게 변경해봤습니다.

File: config.yaml [hunk 2]
```diff
@@ -259,7 +259,91 @@ languages:
...
+      resources:
+        title: 다운로드 + 리소스
+        subtitle: Falco 시작하기, 오픈소스 프로젝트에 컨트리뷰션 등 정보 공유.
```
Comment (jmyung) [inline config.yaml]:
```suggestion
        subtitle: 지금 시작하기, 오픈소스 프로젝트에 컨트리뷰션 등 정보 공유.
```
원문에 가깝게 변경해봤습니다.

File: config.yaml [hunk 3]
```diff
@@ -259,7 +259,91 @@ languages:
...
+        items:
+          - title: 컨트리뷰션
+            text: Github으로 이동해서 오픈소스 Eco-System의 컨트리뷰터가 되세요.
```
Comment (jmyung) [inline config.yaml]:
```suggestion
            text: 깃헙(Github)으로 이동하여 오픈소스 생태계의 컨트리뷰터가 되세요.
```
- 깃헙으로 많이 사용되므로, 한영 병기 표기를 제안드립니다.
- 생태계 표기로 제안드립니다.

File: config.yaml [hunk 4]
```diff
@@ -259,7 +259,91 @@ languages:
...
+            # Icon (from /images/illustrations/icons/___.svg)
+            icon: group.png
+            buttonText:  Falco Github
```
Comment (jmyung) [inline config.yaml]:
```suggestion
            buttonText:  Falco 깃헙
```

File: config.yaml [hunk 5]
```diff
@@ -259,7 +259,91 @@ languages:
...
+            buttonLink: https://github.com/falcosecurity/falco
+          - title: 다운로드
+            text: Falco 설치가이드를 통해 시작하기
```
Comment (jmyung) [inline config.yaml]:
```suggestion
            text: Falco 설치 가이드를 통해 시작하기
```
띄어쓰기가 필요합니다.

File: config.yaml [hunk 6]
```diff
@@ -259,7 +259,91 @@ languages:
...
+            icon: download.png
+            buttonText: Falco 설치가이드
```
Comment (jmyung) [inline config.yaml]:
```suggestion
            buttonText: Falco 설치 가이드
```
띄어쓰기가 필요합니다.

File: config.yaml [hunk 7]
```diff
@@ -259,7 +259,91 @@ languages:
...
+            buttonLink: /docs/installation
+          - title: Documentation
```
Comment (jmyung) [inline config.yaml]:
```suggestion
          - title: 문서화
```
문서화로 많이 사용되고 있습니다.

File: config.yaml [hunk 8]
```diff
@@ -259,7 +259,91 @@ languages:
...
+            text: 공식 Docs를 통해 [Falco 설치하기](/docs/installation), [설정 방법](/docs/configuration), [커스텀 룰](/docs/rules)을 작성하는 방법을 배워보세요.
```
Comment (jmyung) [inline config.yaml]:
```suggestion
            text: 공식 문서를 통해 [Falco 설치하기](/docs/installation), [설정 옵션](/docs/configuration), [사용자 정의 규칙](/docs/rules)을 작성하는 방법을 배워보세요.
```
- custom은 `사용자 정의` 로 표기하는 것으로 의견을 드립니다. 
- 좀 더 원문에 가깝게 변경해봤습니다.

File: config.yaml [hunk 9]
```diff
@@ -259,7 +259,91 @@ languages:
...
+        buttonText: 제출하기
+    menu:
+      navbar:
+        - name: 왜 Falco인가?
+          url: "/#whyfalco"
+          weight: 2
+        - name: 시작하기
+          url: "/#resources"
+          weight: 3
+        - name: Documentation
```
Comment (jmyung) [inline config.yaml]:
```suggestion
        - name: 문서화
```
문서화로 많이 사용되고 있습니다.

File: config.yaml [hunk 10]
```diff
@@ -259,7 +259,91 @@ languages:
       - name: ブログ
         url: "/blog"
         weight: 5
-       
+
+  kr:
+    languageName: 한글 Korean
```
[the remark this answers is not in the document]
Comment (onesolpark) [reply config.yaml]: 넵 변경하겠습니다

File: config.yaml [hunk 11]
```diff
@@ -259,7 +259,91 @@ languages:
...
+    title: Falco — 클라우드 네이티브 런타임 보안(Cloud-Native Runtime Security)
+    contentDir: content/kr
+    params:
+      heroText:
+        # Main hero title
+        title: 클라우드 네이티브 런타임 보안
+        # Hero subtitle (optional)
+        subtitle: Falco, 오픈소스 클라우드 네이티브 런타임 보안 프로젝트는 쿠버네티스의 위협을 감지 표준 엔진입니다. Falco는 런타임시 발생하는 어플리케이션 이상 감지 및 위협에 대한 알림을 지원합니다.
```
[the remark this answers is not in the document]
Comment (onesolpark) [reply config.yaml]: 넵 변경하겠습니다

**Comment by leodido** [comment]
Sure you can review it @jmyung! Thanks 🤗 

As soon this is complete, I'll propose you two for the OWNERS file of the `content/ko` subdirectory of this repository, if you two agree :)

File: README.md [hunk 12]
```diff
@@ -47,3 +47,14 @@ To translate other parts of the site:
 
 1. See the `languages.jp` block in the site's [`config.yaml`](./config.yaml) configuration file and translate those items from the current English.
 1. See [`i18n/jp.yaml`](./i18n/jp.yaml) for a handful of language snippets that need to be translated from the current English. Translate the `other` block of each term.
+
+### Korean
+
+There is currently scaffolding in place for a Korean translation of the site.
+
+To translate docs into Korean, create Korean Markdown file(s) in the `content/kr/docs` directory.
```
Comment (leodido) [inline README.md]: Can we use the ISO 639-1 code for Korean language? Ie., `ko`.

File: content/kr/docs/_index.md [hunk 13]
```diff
@@ -0,0 +1,42 @@
+---
```
Comment (leodido) [inline content/kr/docs/_index.md]: Can we use `ko` as subdirectory?

File: i18n/kr.yaml [hunk 14]
```diff
@@ -0,0 +1,6 @@
+integrations_and_platforms:
```
Comment (leodido) [inline i18n/kr.yaml]: Also this file. Can we use the ISO 639-1 code to refer to Korean language?

It should be `ko`, imho :)

File: README.md (hunk 12 above)
Comment (jmyung) [reply README.md]: @onesolpark
ko is correct ! 😮

Thanks @leodido

**Comment by jmyung** [comment]
> Sure you can review it @jmyung! Thanks 🤗
> 
> As soon this is complete, I'll propose you two for the OWNERS file of the `content/ko` subdirectory of this repository, if you two agree :)

We appreciate it. 👍

File: config.yaml [hunk 15]
```diff
@@ -259,7 +259,91 @@ languages:
       - name: ブログ
         url: "/blog"
         weight: 5
-       
+
+  kr:
```
Comment (leodido) [inline config.yaml:263]: Also, as soon PR #178 gets merged in, the Korean translation of config.yaml file have to be put in its own file under `data/ko/config.yaml`

Comment (onesolpark) [reply config.yaml:263]: got it ill get the changes in soon

**Comment by leodido** [comment]
Any update on this? :)

**Comment by onesolpark** [comment]
I'm sorry this is taking long.
I'll get this through(including the reviews and changes) before the end of this week. :)

**Comment by fntlnz** [comment]
On Wed, Apr 29, 2020 at 03:25:43AM -0700, Han Sol Park wrote:
> I'm sorry this is taking long.
> I'll get this through(including the reviews and changes) before the end of this week. :) 
> 
> -- 
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/falcosecurity/falco-website/pull/177#issuecomment-621114595

Thank you @onesolpark,
we are really excited to have this.

Thanks for the review @jmyung

Lore

**Comment by futureCreator** [comment]
@onesolpark 
You can close this PR 😃 
Korean content has already been added.
👉 PR #243

**Pull request closed without merging**